R/filter_parameters.R

```diff
@@ -80,11 +80,11 @@ filter_by_join <- function(x, parameters = NULL, nm = "") {
   }
   extra_names <- setdiff(filter_names, param_names)
   if (length(extra_names) > 0) {
-    msg <- paste0(
-      "There are unneeded columns in `parameters` that were ignored: ",
-      paste0("'", extra_names, "'", collapse = ", ")
-    )
-    rlang::warn(msg)
+    cli_warn(c(
```

Comment:
I think you don't need the `c()` here

Reply:
You're right! Deleted in [80cfe61](https://github.com/tidymodels/tune/pull/587/commits/80cfe61004a3d4ca6494444181f1554e60e86aed).